parachain/pallets/ethereum-beacon-client/src/lib.rs

```diff
@@ -1116,6 +1082,50 @@ pub mod pallet {
 				},
 			}
 		}
+
+		pub fn instantiate_milagro_pubkeys(
```

Comment:
Lets describe the actual operation being performed
```suggestion
		pub fn decompress_pubkeys(
```

Reply:
also these functions need comments.

Reply:
It's actually operation for decompress&deserialize to G1 points so IMHO `instantiate` could be more accurate here.